validator/pom.xml

```diff
@@ -175,18 +182,33 @@
                 <artifactId>maven-assembly-plugin</artifactId>
                 <configuration>
                     <archive>
+                        <index>true</index>
+                        <manifest>
+                            <Main-Class>org.odftoolkit.odfvalidator.Main</Main-Class>
```

Comment:
this change isn't related to what's in the commit message

Reply:
Do not be so picky, it is the same as the previous, starting the WAR by JAR command line parameter